### PR TITLE
add a section to FAQ & fix a little error in `README.md`

### PR DESCRIPTION
**Pull request by gcanoxl**

### CODE REVIEW

**Review by jcs090218 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -232,6 +232,10 @@ lsp-dart supports running Flutter and Dart commands as following:
 
 :small_blue_diamond: Try to set the `lsp-dart-sdk-dir` to the Dart SDK dir installation or if you are using Flutter, `<your-flutter-dir>/bin/cache/dart-sdk/`.
 
+:grey_question: `dap-debug` couldn't run the app on advices or white screen.
+
+:small_blue_diamond: Try to set `export NO_PROXY=localhost,127.0.0.1` into your `.bash_profile` file.
```
Comment (ericdallo) [inline README.md:237]: Maybe link that issue from dart code?

File: README.md [hunk 2]
```diff
@@ -232,6 +232,10 @@ lsp-dart supports running Flutter and Dart commands as following:
 
 :small_blue_diamond: Try to set the `lsp-dart-sdk-dir` to the Dart SDK dir installation or if you are using Flutter, `<your-flutter-dir>/bin/cache/dart-sdk/`.
 
+:grey_question: `dap-debug` couldn't run the app on advices or white screen.
```
Comment (ericdallo) [inline README.md:235]: Did you mean?
```suggestion
:grey_question: `dap-debug` couldn't run the app on devices or shows a white screen.
```

Comment (gcanoxl) [reply README.md:235]: hahaha yes!

Comment (gcanoxl) [reply README.md:235]: thanks for the correction

File: README.md (hunk 1 above)
Comment (gcanoxl) [reply README.md:237]: I think that if we have already known how to solve a problem, we should just tell users as terse as we could lest they haven't got enough patience to read.

**Pull request closed without merging**